candle-nn/src/embedding.rs

```diff
@@ -18,6 +18,11 @@ impl Embedding {
     pub fn embeddings(&self) -> &Tensor {
         &self.embeddings
     }
+
+    /// Get the hidden size of the embedding matrix
+    pub fn hidden_size(&self) -> usize {
+        self.embeddings.dim(1).unwrap()
```

Comment:
Expose `self.hidden_size` instead? This would avoid the `unwrap` (better having as few of these as possible)

Reply:
True, I will convert that.

Reply:
Thanks